crates/cargo-contract/src/main.rs

```diff
@@ -71,6 +72,12 @@ use assert_cmd as _;
 #[cfg(test)]
 use predicates as _;
 
+#[cfg(test)]
+use regex as _;
+
+// Only used on windows.
+use which as _;
```

Comment:
Is it not possible to prefix with `#[cfg(target_os = "windows")]` here?

Reply:
I don't think so. This is intentionally doing the opposite - making it look like `which` is used on not-windows. `cargo-contract` has a clippy lint turned on to check that linked crates are actually used, and `which` is only used on windows, so clippy doesn't see it on unix. What could be done is to change the `Cargo.toml` to only link to `which` on windows.